packages/open-api-page/docs.mdx

```diff
@@ -0,0 +1,65 @@
+---
+componentName: 'OpenApiPage'
+---
+
+The `<OpenApiPage />` component, and related server utilities, are used to render documentation pages for a provided [Swagger OpenAPI 2.0 specification](https://swagger.io/specification/v2/).
+
+<LiveComponent
```

Comment:
This is the best I could come up with for displaying these fixtures. Involves some kind-of-annoying work to generate these fixtures, but figured this is acceptable for now. Very open to ideas on how we could better support these kinds of use cases in Swingset in the future 🙇‍♂️